src/aap_eda/api/views/project.py

```diff
@@ -171,38 +171,19 @@ def create(self, request):
         },
     )
     def retrieve(self, request, pk):
-        project = super().retrieve(request, pk)
```

Comment:
mixins.RetrieveModelMixin is no longer necessary in ProjectViewSet after this change, right?